Upgrade-Guide/op_version.md

```diff
@@ -0,0 +1,29 @@
+
+###op-version
+op-version is the operating version of the Gluster which is running.
+
+After Gluster upgrade, it is important that op-version is updated in all peers to avoid any 
+compatibility issues.
+
+###Updating op-version
+
+Once Gluster is upgraded, ensure that the op-version is also updated.
+
+    [root@~]#cat /var/lib/glusterd/vols/tv1/info  | grep ^op-version
+
+
+If this is not done automatically set op-version as below.
+For example, after upgrading to glusterfs-3.7.1, set op-version as:
+
+    [root@~]#gluster volume set all cluster.op-version 30701
+
+### Understanding op-version
+
+The op-version for a given X.Y.Z release will be an integer XYZ, with
```

Comment:
@kshlm @atinmu  << But I don't really if it is done every release.. I think we can have it incremented for every  version update - as this will make things consistent and clear (Only thing is this needs to tracked such that version increment means op-version increment)